documentation/bolt_configure_orchestrator.md

```diff
@@ -1,13 +1,16 @@
 # Connecting Bolt to Puppet Enterprise
 
-If you're a Puppet Enterprise (PE) customer, you can connect Bolt to Puppet Enterprise (PE) using the
-Puppet Communications Protocol (PCP) transport. However, in most cases this is not
-necessary, because tasks and plans are already supported from the console or the
-command line using
-[PE orchestrator](https://puppet.com/docs/pe/latest/running_jobs_with_puppet_orchestrator_overview.html). Wherever possible, we recommend using PE tasks and plans instead of connecting
+If you're a Puppet Enterprise (PE) customer, you can connect Bolt to Puppet
+Enterprise (PE) using the Puppet Communications Protocol (PCP) transport.
+However, in most cases this is not necessary, because tasks and plans are
+already supported from the console or the command line using
+[PE orchestrator](https://puppet.com/docs/pe/latest/running_jobs_with_puppet_orchestrator_overview.html).
+Wherever possible, we recommend using PE tasks and plans instead of connecting
 Bolt to PE over PCP. 
 
 For more information on tasks and plans in PE, see [Tasks and
 plans](https://puppet.com/docs/pe/latest/running_tasks_and_plans_pe.html).
 
-For information on connecting Bolt to PE, see [Connecting Bolt to PE](https://puppet.com/docs/pe/latest/bolt_configure_orchestrator.html).
\ No newline at end of file
+For information on connecting Bolt to PE using the `bolt-shim` module, see
+[Connecting Bolt to
+PE](https://github.com/puppetlabs/puppetlabs-bolt_shim/blob/master/docs/connect_bolt_pe.md).
```

Comment:
Ah, does this not get published to the docs site anywhere?

Reply:
Not any more. I had to move the doc out of PE cos the module is not officially supported, and documenting it on the site gives users the impression that it _is_ supported. I contemplated removing our doc as well, but I think sending people off to a git repo sends a pretty good message that it's an "At your own risk" situation.